### PR TITLE
Update ErrorView Guide

### PR DESCRIPTION
**Pull request by iv-mexx**
The guide does not match with the default `ErrorView` of a new Project anymore.

I tried to update the guide to match the current generated ErrorView. Since I'm just learning Elixir and Phoenix, I hope I understood everything correctly.

 I'm not sure if its necessary to mention implementing the `def render("404.html", _assigns)` function because

a) It is deleted in the next step right away and does not provide much value over the the default implementation
b) `ErrorView` already contains an example of this possibility which, in my opinion, is very self explanatory

```
  # If you want to customize a particular status code
  # for a certain format, you may uncomment below.
  # def render("500.html", _assigns) do
  #   "Internal Server Error"
  # end
```

### CODE REVIEW

File: guides/views.md [hunk 1]
```diff
@@ -212,19 +217,35 @@ config :hello, HelloWeb.Endpoint,
...
 
 The first question is, where does that error string come from? The answer is right in the `ErrorView`.
 
+```elixir
+def template_not_found(template, _assigns) do
+  Phoenix.Controller.status_message_from_template(template)
+end
+```
+
+Great, so we have this `template_not_foun/2` function that takes a template and an `assigns` map, which we ignore. Where is this function being called from? The answer is the `render/5` function defined in the `Phoenix.Endpoint.RenderErrors` module. The whole purpose of this module is to catch errors and render them with a view, in our case, the `HelloWeb.ErrorView`.
```
Comment (Gazler) [inline guides/views.md]: `template_not_foun` is missing a "d".

**Comment by Gazler** [comment]
Thanks! One small change then I think we can merge this in.

File: guides/views.md (hunk 1 above)
Comment (iv-mexx) [reply guides/views.md]: Done! 👍

**Comment by Gazler** [comment]
Thanks <3

**Pull request merged**